src/frontend/app/features/cloud-foundry/edit-space/edit-space-step/edit-space-step.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="edit-space">
```

Comment:
There's a whole lot of copy and paste here between add/edit. These should probably be merged into a single addEdit component with different add/edit modes

Reply:
I've refactored out common code b/w the AddSpaceStepComponent and the EditSpaceStepComponent into an abstract class which both extend.